jena-arq/src/main/java/org/apache/jena/sparql/engine/main/iterator/QueryIterService.java

```diff
@@ -32,8 +32,8 @@
 import org.apache.jena.sparql.engine.main.QC ;
 import org.apache.jena.sparql.exec.http.Service;
 import org.apache.jena.sparql.service.ServiceExecution;
```

Comment:
This is now an unused import and can be removed.

Reply:
I removed the import and added a deprecation annotation because QueryIterService is now no longer used but existing code may have sub-classed from it.